Code/Framework/AzFramework/Platform/iOS/AzFramework/Input/Devices/Gamepad/InputDeviceGamepad_iOS.mm

```diff
@@ -68,23 +67,28 @@
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 namespace AzFramework
 {
+    static constexpr int GetMaxIosSupportedGamepads()
+    {
+        return static_cast<int>(GCControllerPlayerIndex4) + 1;
+    }
+
     ////////////////////////////////////////////////////////////////////////////////////////////////
     //! Platform specific implementation for apple game-pad input devices
-    class InputDeviceGamepadApple : public InputDeviceGamepad::Implementation
+    class InputDeviceGamepadiOS : public InputDeviceGamepad::Implementation
```

Comment:
Not sure I understand the renaming of GamepadApple -> GamepadiOS? Are gamepads really supported on iOS? I would think this would be for gamepads connected to Macs?

Reply:
I'm not sure, this use to be a shared file between both iOS and Mac, and I don't have the context on why gamepad is on iOS. I removed the sharing of files.